### ISSUE TITLE
Proper way to stop consul-alerts service

### ISSUE THREAD

**Issue by ArturChe** [AcalephStorage/consul-alerts#227]
Hi all!

I have a question about a proper way to stop consul-alerts service.
Background: to start consul-alerts service I use sysvinit bash script where consul-alerts starts as a daemon.

So the question is how to stop consul-alerts service properly. Is there any command that I should send to service (for example consul has 'leave' command). Or I can just kill the process by id and clean pid file?

**Comment by gneill794** [comment]
I see this question from from march.   I have the same question, with the addition of how to properly restart/reload.   I am trying to write a proper systemd unit file.  ArturChe, did you get anywhere with this?

**Comment by eripa** [comment]
Sending `SIGTERM` does the trick. This is the standard in systemd and most other process managers.

```
consul-alerts[7661]: time="2018-06-01T20:52:44Z" level=info msg="Running health check."
consul-alerts[7661]: time="2018-06-01T20:52:58Z" level=info msg="Shutting down..."
consul-alerts[7661]: time="2018-06-01T20:52:58Z" level=info msg="cleaning up"
consul-alerts[7661]: time="2018-06-01T20:52:58Z" level=info msg="Lost leadership."
consul-alerts[7661]: time="2018-06-01T20:52:58Z" level=info msg="cleanup done"
```

**Comment by ArturChe** [comment]
Yep, that's right. In source code I have fond that service can properly process SIGTERM and SIGKILL signals
